spec.go

```diff
@@ -142,10 +142,10 @@ func parseIDMap(idMap string, uid, gid, size *uint32) error {
 	*gid = uint32(num[1])
 	*size = uint32(num[2])
 
-	if *uid > math.MaxUint32 || *gid > math.MaxUint32 || *size < syscont.IdRangeMin {
-		return fmt.Errorf("invalid id-map \"%v\": uid and gid must be <= %v, size must be >= %v",
-			idMap, math.MaxUint32, syscont.IdRangeMin)
-	}
+//	if *uid > math.MaxUint32 || *gid > math.MaxUint32 || *size < syscont.IdRangeMin {
```

Comment:
Why is this commented?

Reply:
Good catch, I forgot to go back to to this.

This block fails to compile for 32bit. Something about `math.MaxUint32` being too large for `%v`. A quick Google indicated that `%v` should basically be able to print any type, so I'm not sure how to resolve the error--not being a go programmer myself.

Any thoughts?

Reply:
Ah, strange, `math.MaxUint32` is just a regular `1<<32 - 1`, please try `%d` instead.

Reply:
I was able to resolve this compile error using the advise here: https://stackoverflow.com/questions/36231049/why-does-printing-out-max-int-cause-compile-error-in-golang